### DISCUSSION TITLE
Dapr Workflow Integration

### DISCUSSION THREAD

**Discussion by tomfanara** [serverlessworkflow/specification#762] [General]
Hello has anyone integrated another code based workflow SDK such as Dapr Workflows? So when declaratively building a workflow the JSON or YAML could produce a templated approach to writing Dapr workflows (operations) and activities (events)

**Comment by ricardozanini** [comment]
Hi @tomfanara! I've heard them talking about implementing workflows, but I don't know anything else related to this topic. Based on a guess, I'd say that should be reasonable to write a template/conversor. What do you mean by:

> So when declaratively building a workflow the JSON or YAML could produce a templated approach to writing Dapr workflows (operations) and activities (events)

Not quite sure if I understood.

**Comment by ricardozanini** [comment]
Hi! I'm closing this one due to inactivity. Please feel free to reopen if you still have questions or need more clarification.
